src/lib/yang/alarms.lua

```diff
@@ -581,6 +578,27 @@ function CounterAlarm:check ()
    end
 end
 
+ExprAlarm = {}
```

Comment:
Note for followup: we should call this a "CallbackAlarm" instead.  ExprAlarm is too imprecise I think.